trio/_threads.py

```diff
@@ -27,13 +27,14 @@
 from ._util import coroutine_or_error
 
 RetT = TypeVar("RetT")
-Ret2T = TypeVar("Ret2T")
 
 
 class _TokenLocal(threading.local):
     """Global due to Threading API, thread local storage for trio token."""
 
     token: TrioToken
+    cancel_register: list[RaiseCancelT | None]
+    task_register: list[trio.lowlevel.Task | None]
 
 
 TOKEN_LOCAL = _TokenLocal()
```

Comment:
it's not just the TrioToken anymore, maybe worth a new name? first that comes to mind is `ThreadParentData`